src/Clients/IAppClient.cs

```diff
@@ -48,6 +48,18 @@ public interface IAppClient
         /// <remarks>https://getstream.io/chat/docs/dotnet-csharp/push_test/?language=csharp</remarks>
         Task<AppCheckSqsResponse> CheckSqsPushAsync(AppCheckSqsRequest checkSqsRequest);
 
+        /// <summary>Inserts or creates a new push provider.</summary>
+        /// <remarks>https://getstream.io/chat/docs/dotnet-csharp/push_introduction/?language=csharp</remarks>
+        Task<UpsertPushProviderResponse> UpsertPushProviderAsync(PushProviderRequest pushProviderRequest);
+
+        /// <summary>Lists all push providers.</summary>
+        /// <remarks>https://getstream.io/chat/docs/dotnet-csharp/push_introduction/?language=csharp</remarks>
+        Task<ListPushProvidersResponse> ListPushProvidersAsync();
+
+        /// <summary>Deletes a push provider.</summary>
+        /// <remarks>https://getstream.io/chat/docs/dotnet-csharp/push_introduction/?language=csharp</remarks>
+        Task<ApiResponse> DeletePushProviderAsync(string providerType, string name);
```

Comment:
good to type the `Type`

Reply:
`type` is a keyword in C#, that's why I went for `providerType`. same for golang

Reply:
I mean using a type instead of string

Reply:
✅